src/pages/nft/claim/index.vue

```diff
@@ -319,26 +511,39 @@ import {
   getNFTClassCollectionType,
   nftClassCollectionType,
   parseNFTMetadataURL,
-  LIKE_ADDRESS_REGEX,
 } from '~/util/nft';
+import { ellipsis } from '~/util/ui';
 
 import alertMixin from '~/mixins/alert';
 import walletMixin from '~/mixins/wallet';
 import nftMixin from '~/mixins/nft';
 import nftOrCollectionMixin from '~/mixins/nft-or-collection';
+import walletLoginMixin from '~/mixins/wallet-login';
+import collectionMixin from '~/mixins/nft-collection';
 
 const NFT_CLAIM_STATE = {
-  GIFTING: 'GIFTING',
-  INITIAL: 'INITIAL',
+  WELCOME: 'WELCOME',
+  LOGIN: 'LOGIN',
+  ID_CONFIRMATION: 'ID_CONFIRMATION',
+  MESSAGE: 'MESSAGE',
   CLAIMING: 'CLAIMING',
   CLAIMED: 'CLAIMED',
-  PHYSICAL_ONLY: 'PHYSICAL_ONLY',
   ERROR: 'ERROR',
 };
 
 export default {
   name: 'NFTClaimPage',
-  mixins: [alertMixin, walletMixin, nftMixin, nftOrCollectionMixin],
+  filters: {
+    ellipsis,
+  },
+  mixins: [
+    alertMixin,
+    walletMixin,
+    nftMixin,
+    nftOrCollectionMixin,
+    walletLoginMixin,
+    collectionMixin,
```

Comment:
The ordering should be `nft, collection, nftOrCollection`? Since last one override previous ones

Reply:
[🎨 Prevent nftOrCollectionMixin being overwritten](https://github.com/likecoin/liker-land/pull/1588/commits/441de8b43dc4ac98b9a2543a480334f963df3ca5)